sycl/test-e2e/Matrix/get_coord_int8_matB.cpp

```diff
@@ -5,14 +5,11 @@
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 //
 //===----------------------------------------------------------------------===//
-// REQUIRES: matrix
+// REQUIRES: aspect-ext_intel_matrix
 
 // RUN: %{build} -o %t.out
 // RUN: %{run} %t.out
 // XFAIL: cpu
```

Comment:
this test requires newer igc to pass on DG2. It was marked as xfail:* in XMX8 folder. I'm wondering, case we use something like below to make it expected fail on DG2?
```suggestion
// XFAIL: cpu, gpu-intel-dg2
```

Reply:
This change SGTM